server/routes/organizations/employees/employee-controller.js

```diff
@@ -1,29 +1,79 @@
+var models = require('../../../db/models');
```

Comment:
Yeah so we need some way to avoid creating multiple connections. You can monitor what the database is doing with opening the mysql command line tools and executing: 'show status'

If you want to count the number of connections you can do: 'show status like '%onn%';

I tried exporting app from server.js and importing it and then doing app.get but it was throwing an error. This will work fine for now, we can figure out why the app.get method isnt working at a later time